pkg/config/client.go

```diff
@@ -0,0 +1,93 @@
+package config
```

Comment:
could we name this package as mulitcluster or something else more relevant rather than just config

Reply:
FYI this is related to the group, ie: config.openservicemesh.io. So this would be the "config" client for all the services in that group. Not sure if that changes your comment